app/views/shared/_submit_buttons.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="govuk-button-group">
+  <%= form.submit(
```

Comment:
For some reason, although I can't see anything about the gov uk form builder docs, `form.govuk_submit` renders a button where the text is a before pseudo-element, and therefore `params[:commit]` is nil 🤯